shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-distsql/shardingsphere-readwrite-splitting-distsql-statement/src/main/java/org/apache/shardingsphere/readwritesplitting/distsql/parser/segment/ReadwriteSplittingRuleSegment.java

```diff
@@ -58,7 +58,7 @@ public ReadwriteSplittingRuleSegment(final String name, final String writeDataSo
      * 
      * @return is auto ware or not
```

Comment:
`ware` or `aware`?

Reply:
Sry,  it is adjusted later in pr, because this pr is more anxious, but the time of ci is too long

Reply:
It's solved in this  https://github.com/apache/shardingsphere/pull/14967